ThreadGroup/Views/TGSelectDeviceStepView.m

```diff
@@ -65,6 +65,10 @@ - (void)commonInit {
 
 #pragma mark - Public
 
+- (void)setTopBarHidden:(BOOL)hidden {
+    self.topBar.hidden = hidden;
+}
```

Comment:
Ah...but its done in TGDeviceStepView because we make the method public instead of the property.